pkg/storage/streams/store.go

```diff
@@ -187,32 +213,53 @@ func (s *streamStore) Put(ctx context.Context, path paths.Path, data io.Reader,
 				Metadata:                 metadata,
 				EncryptionType:           int32(s.encType),
 				EncryptionBlockSize:      int32(s.encBlockSize),
-				LastSegmentEncryptionKey: encryptedEncKey,
+				LastSegmentEncryptionKey: encKeyByte,
 			}
+			
 			lastSegmentMeta, err := proto.Marshal(&msi)
 			if err != nil {
 				return nil, nil, err
 			}
-			return lastSegmentPath, lastSegmentMeta, nil
-		})
+
+			logger.Debug("WE ARE IN  put STREAMSTORE -1\n\n\n\n\n\n")
+			fmt.Println("this is the marshalled last segmentMeta: ", lastSegmentMeta, "\n\n\n")
```

Comment:
Nit: remove comments

Reply:
yup! it's a WIP- not ready yet (just finished working on a bug) should be ready now